client/src/Components/JournalEntry/JournalEntry.js

```diff
@@ -1,12 +1,48 @@
-import React from "react";
+import React, { useState } from "react";
 import { Link } from "react-router-dom";
 import { DateTime } from 'luxon';
 import './JournalEntry.css';
+import { updateEntry } from "../../utilities/api-calls";
 
-const JournalEntry = ({ journal }) => {
+const JournalEntry = ({ journal, entryID }) => {
 
-  const {date, feeling, activity, journal_entry} = journal;
-  const formattedDate = DateTime.fromFormat(date, 'yyyy-MM-dd')
+  const {date, feeling, activity, journal_entry } = journal;
+
+  const [entry, setEntry] = useState(journal_entry);
+  const [savedEntry, setSavedEntry] = useState(journal_entry);
```

Comment:
nice! love a good controlled form

Reply:
Love seeing all the components tied together into one value!